demos/storybook/.storybook/preview-head.html

```diff
@@ -10,6 +10,5 @@
     flex-direction: column;
     align-items: center;
     justify-content: center;
-    background: #efefef;
```

Comment:
Can we preserve the slight offwhite background for the canvas?  I really think it helps the components standout for light theme.  

![image](https://user-images.githubusercontent.com/6538289/78674833-36f8ac80-78b2-11ea-8d89-24876617d98b.png)


vs 

![image](https://user-images.githubusercontent.com/6538289/78675049-8212bf80-78b2-11ea-9249-e4df6a542582.png)

Reply:
I agree with Evan.  The off-white (darker) background makes the component standout in light theme.

Reply:
Setting the background-color here overrides the theme so when we go to dark mode that canvas color persists. Could possibly use javascript to apply the styles based on the theme if we really want this.

Reply:
Maybe do an inline style for light theme.

Reply:
Or we can use theme.palette.background.default for dark and light theme respectively. Gray50 for light and BlackA200 for dark. 

Gray50 is a typical color used when there are some UI elements that need to "pop-out" and demands a higher elevation. Specially card-like elements.